convert_matrices.py

```diff
@@ -168,8 +172,6 @@ def _copy_matrices(self, *matrices: Matrix):
                     idempotent_link(src, dst)
 
     def _convert_matrices(self, *inputs: Union[CSV, H5, CSVPerCell]):
-        names = [input_.name for input_ in inputs]
-        assert len(names) == len(set(names))
```

Comment:
Why does this need to be removed?

Reply:
My mistake, I removed the `name` field from `CSVPerCell` and forgot that I re-added it as a property. This should stay.